internal/resources/data_source_lb_monitors.go

```diff
@@ -0,0 +1,113 @@
+// (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+
+package resources
+
+import (
+	"context"
+
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/internal/resources/validations"
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/internal/utils"
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/pkg/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func LBMonitorData() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Computed:    true,
```

Comment:
name is also Computed field ? it is not given by user ?

Reply:
done